productImport/product/productHandler.go

```diff
@@ -29,6 +30,9 @@ func (ph *ProductHandler) InitSourceData(sourceFeedPath string) ([]map[string]in
 		return nil, err
 	}
 	res := parse(sourceData)
+	if len(res) == 0 {
+		return nil, fmt.Errorf("empty source data")
```

Comment:
It's not clear why empty import source interpreted as error. 
As for me it's just NOP.

Reply:
it will be interpreted as valid if it will have header.